server/src/com/gip/xyna/xprc/xsched/xynaobjects/Now.java

```diff
@@ -279,15 +279,42 @@ public Now instance() {
 
   private static void init(Now instance) {
     long now = System.currentTimeMillis();
-    String format = instance.getFormatInternally();
+    String format = instance.getFormatInternally() == null ? DEFAULT_FORMAT.getFormat() : instance.getFormatInternally();
     instance.unversionedSetDate(instance.getOrCreateLazyDateFormat().format(now, format));
   }
+  
+  @Override
+  public void setFormat(DateFormat format) {
+    if(getFormatInternally() == null) {
+      try {
+        getOrCreateLazyDateFormat().validate(getDate(), DEFAULT_FORMAT.getFormat());
+        super.setFormat(DEFAULT_FORMAT);
+      } catch(Exception e) {
+        //no format specified and default format is not valid for current date.
```

Comment:
why should we do nothing in this case? Do this happen often?
It is expansive to build a stacktrace, to just throw it away.